pallets/aggregate/src/data.rs

```diff
@@ -267,6 +272,7 @@ impl<
         Self: MaxEncodedLen,
         BoundedVec<StatementEntry<A, B>, VecSize<S>>: MaxEncodedLen,
         StatementEntry<A, B>: MaxEncodedLen,
+        Destination<B>: MaxEncodedLen,
     {
         let upper = Self::max_encoded_len();
         let aggregation_size =
```

Comment:
With this implementation you're bonding the same amount even if you have a hyperbridge destination or not. I know that now You didn't use a enum for destinations but you can always implement it by remove the max_encoded_len() a add the destination encoded size. In this way it will works well also when you have an enum. Your tests should add a case where you have different destination and compute different sizes